temba/settings_common.py

```diff
@@ -1404,3 +1404,7 @@ def traces_sampler(sampling_context):  # pragma: no cover
 
 CORS_ALLOW_ALL_ORIGINS = True
 CORS_ALLOW_METHODS = ["GET"]
+
+# Contacts import via excel
+# if set to True will not raise error on duplicate, instead will use last row
+IGNORE_DUPLICATES_CONTACT_IMPORT = False
```

Comment:
Can we have this value getting from an environment variable like [here](https://github.com/greatnonprofits-nfp/rapidpro/blob/release/3.4/temba/settings_common.py#L1402)?

Reply:
And I think, for better understanding, would be good to rename this variable to `ALLOW_DUPLICATES_CONTACT_IMPORT` and the default value of it as `True`, and apply the logic on `if urn in seen_urns` statement based on this new name. What do you think?